### PR TITLE
[] hw5 workflow

### PR DESCRIPTION
**Pull request by dthmrphn**

### CODE REVIEW

File: hw5_codegen/handlers_gen/methods_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+type testCase struct {
+	name string
+	mhs  []*MethodHandler
+}
+
+var tests []testCase
+
+func init() {
+	tests = []testCase{
```
Comment (uladzislaudd) [inline hw5_codegen/handlers_gen/methods_test.go:17]: u dont have to use init() here

File: hw5_codegen/handlers_gen/methods.go [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+			}
+			mm[mh.rec] = append(mm[mh.rec], *mh)
+		}
+	}
+
+	if len(mm) == 0 {
+		return nil, fmt.Errorf("no marked methods to generate api")
+	}
+
+	mhs := make([]*MethodHandler, 0)
```
Comment (uladzislaudd) [inline hw5_codegen/handlers_gen/methods.go:277]: `mhs := make([]*MethodHandler, len(mm))`

File: hw5_codegen/handlers_gen/go.mod [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module localhost/coursera/hw5_codegen/handlers_gen
```
Comment (uladzislaudd) [inline hw5_codegen/handlers_gen/go.mod:1]: one go.mod for repo is enough

**Review by uladzislaudd — APPROVED** [review]
LGFM!

File: hw5_codegen/handlers_gen/methods.go (hunk 2 above)
Comment (dthmrphn) [reply hw5_codegen/handlers_gen/methods.go:277]: fixed

File: hw5_codegen/handlers_gen/methods_test.go (hunk 1 above)
Comment (dthmrphn) [reply hw5_codegen/handlers_gen/methods_test.go:17]: gotta do that for comparison benchmarks

File: hw5_codegen/handlers_gen/go.mod (hunk 3 above)
Comment (dthmrphn) [reply hw5_codegen/handlers_gen/go.mod:1]: removed

**Pull request merged**